apps/api/src/controllers/v1/batch-scrape.ts

```
@@ -0,0 +1,173 @@
+  if (urls.length > 1000) {
+    // set base to 21
+    jobPriority = await getJobPriority({
+      plan: req.auth.plan,
+      team_id: req.auth.team_id,
+      basePriority: 21,
+    });
+  }
+  logger.debug("Using job priority " + jobPriority, { jobPriority });
+
+  const scrapeOptions: ScrapeOptions = { ...req.body };
+  delete (scrapeOptions as any).urls;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)